### PR TITLE
Add wpxr-to-static to list of migration tools

### PR DESCRIPTION
**Pull request by danielfdickinson**
Now that I have some documentation for wpxr-to-static (a Python3 program for converting from WordPress to Hugo), I hope you'll agree it's a useful addition to the list of migration tools.

### CODE REVIEW

**Comment by danielfdickinson** [comment]
Any comments or issues, or just busy?

**Comment by danielfdickinson** [comment]
Ping? @gohugoio

**Comment by danielfdickinson** [comment]
@jmooring I'm guessing this hasn't been merged because it's a rare tool and you are reticent to merge random conversion methods. Since it was written to replace ``exitwp`` which was written in Python 2 and is defunct, would you prefer I remove ``exitwp`` rather than adding ``wpxr-to-static``?

**Comment by jmooring** [comment]
We can add this, or replace `exitwp-for-hugo`, but I suspect both will be removed at some point:
<https://github.com/gohugoio/hugoDocs/issues/1573>

I think things like this are better suited to "how to" articles on the forum, or an article somewhere else. 

Your thoughts?

**Comment by danielfdickinson** [comment]
I'll add my thoughts to the linked issue, since I agree and think that the tools pages should be a link to a forum 'wiki' post (or just a discussion).

File: content/en/tools/migrations.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Alternatively, you can use the new [Jekyll import command](/commands/hugo_import
 - [exitwp-for-hugo](https://github.com/wooni005/exitwp-for-hugo) - A python script which works with the xml export from WordPress and converts WordPress pages and posts to Markdown and YAML for hugo.
```
Comment (jmooring) [inline content/en/tools/migrations.md:50]:
```suggestion
```

**Review by jmooring — CHANGES_REQUESTED** [review]

File: content/en/tools/migrations.md (hunk 1 above)
Comment (danielfdickinson) [reply content/en/tools/migrations.md]: I've made the suggested update and also updated the wpxr-to-static text, as it mentioned exitwp-to-hugo and said 'see above'.

File: content/en/tools/migrations.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ Alternatively, you can use the new [Jekyll import command](/commands/hugo_import
...
 - [blog2md](https://github.com/palaniraja/blog2md) - Works with [exported xml](https://en.support.wordpress.com/export/) file of your free YOUR-TLD.wordpress.com website. It also saves approved comments to `YOUR-POST-NAME-comments.md` file along with posts.
 - [wordhugopress](https://github.com/nantipov/wordhugopress) - A small utility written in Java, exports the entire WordPress site from the database and resource (e.g. images) files stored locally or remotely. Therefore, migration from the backup files is possible. Supports merging of the multiple WordPress sites into a single Hugo one.
+- [wpxr-to-static](https://github.com/danielfdickinson/wpxr-to-static) - WordPress eXtended RSS to Static Generator Conversion, Python3 program to convert WordPress XML Export (WPXR) format files to Hugo Markdown files (with YAML metadata) and a ``config.toml`` for the site. Inspired by [exitwp-for-hugo](https://github.com/wooni005/exitwp-for-hugo) (a Python 2 \[EOL] script which works with the xml export from WordPress and converts WordPress pages and posts to Markdown and YAML for hugo).
```
Comment (jmooring) [inline content/en/tools/migrations.md:52]:
```suggestion
- [wpxr-to-static](https://github.com/danielfdickinson/wpxr-to-static) - WordPress eXtended RSS to Static Generator Conversion, Python3 program to convert WordPress XML Export (WPXR) format files to Hugo Markdown files (with YAML metadata) and a `config.toml` for the site. Inspired by [exitwp-for-hugo](https://github.com/wooni005/exitwp-for-hugo) (a Python 2 \[EOL] script which works with the xml export from WordPress and converts WordPress pages and posts to Markdown and YAML for hugo).
```

**Review by jmooring — CHANGES_REQUESTED** [review]

File: content/en/tools/migrations.md (hunk 2 above)
Comment (danielfdickinson) [reply content/en/tools/migrations.md]: I assume you would squash merge, or should I squash?

**Pull request merged**